src/main/java/dataMgmt/models/ProjectMetadata.java

```diff
@@ -1,33 +1,41 @@
 package dataMgmt.models;
 
 import com.fasterxml.jackson.annotation.JsonProperty;
+import websocket.models.Permission;
 
 import java.util.Arrays;
+import java.util.HashMap;
 
 /**
  * The structure for project metadata read from the project's root.
  * Created by fahslaj on 5/2/2016.
  */
 public class ProjectMetadata {
-
     @JsonProperty("ProjectID")
-    protected long projectId;
+    private long projectID;
 
     @JsonProperty("Name")
-    protected String name;
+    private String name;
+
+    @JsonProperty("Permissions")
+    private HashMap<String, Permission> permissions;
 
-    @JsonProperty("Owner")
-    protected String owner;
+    // Used in User.Projects - needs to use permissions eventually.
```

Comment:
This is a copied comment from the Websocket.models.project object; User.Project currently uses a permissionlevel, but should return a permission map. This relates to the card that I think you were pulling off the server backlog.